scripts/cnode-helper-scripts/ogmios.sh

```diff
@@ -37,6 +38,19 @@ set_defaults() {
   [[ -z "${OGMIOSBIN}" ]] && OGMIOSBIN="${HOME}"/.local/bin/ogmios
   [[ -z "${HOSTADDR}" ]] && HOSTADDR=127.0.0.1
   [[ -z "${HOSTPORT}" ]] && HOSTPORT=1337
+  if [[ -z "${LOG_LEVEL}" ]]; then
+    LOG_LEVEL=Notice
+  else
+    case ${LOG_LEVEL} in
+      Debug)   : ;;
+      Info)    : ;;
+      Notice)  : ;;
```

Comment:
can remove this line  (default already at notice)?

Reply:
The case statement is there to "accept" all valid entries and default to set log level to Notice if invalid log level is set in user config section. There needs to be a correct log level set else the execution argument will fail. The other option would be to put entire thing in variable, ie including `--log-level`, but not sure if that adds any value?